pkg/cli/admin/ocpcertificates/regeneratemco/rotatecerts.go

```diff
@@ -120,3 +124,38 @@ func (o *RegenerateMCOOptions) Run(ctx context.Context) error {
 	}
 	return nil
 }
+
+func getServerIPsFromInfra(cfg *configv1.Infrastructure) []string {
+	if cfg.Status.PlatformStatus == nil {
+		return []string{}
+	}
+	switch cfg.Status.PlatformStatus.Type {
+	case configv1.BareMetalPlatformType:
+		if cfg.Status.PlatformStatus.BareMetal.APIServerInternalIPs != nil {
+			return cfg.Status.PlatformStatus.BareMetal.APIServerInternalIPs
+		}
+		return []string{cfg.Status.PlatformStatus.BareMetal.APIServerInternalIP}
```

Comment:
As it turns out our BM IPI QE installs only have this (legacy?) field. Just to make triple sure, all platforms will fall back to IP and not IPs